packages/react/src/components/CardEditor/CardEditForm/CardEditForm.jsx

```diff
@@ -143,209 +126,59 @@ export const getCardSizeText = (size, i18n) => {
   return `${sizeName} ${sizeDimensions}`;
 };
 
-/**
- * Returns errors of basic JSON syntax
- * Must not be empty string, Must be valid JSON
- * @param {Object} card JSON currently being edited
- * @returns {Array<string>} error strings
- */
-export const basicCardValidation = (card) => {
-  const errors = [];
-  try {
-    const json = JSON.parse(card);
-    if (!json || typeof json !== 'object') {
-      errors.push(`${card.substring(0, 8)} is not valid JSON`);
-    }
-  } catch (e) {
-    errors.push(e.message);
-  }
-  return errors;
-};
-
-/**
- * removes properties needed for the editor that we don't want the user to be able to modify
- * @param {Object} card JSON currently being edited
- * @returns {Object} card with omitted attributes
- */
-export const hideCardPropertiesForEditor = (card) => {
-  let attributes;
-  let series;
-  let columns;
-  if (card.content?.attributes) {
-    attributes = card.content.attributes.map((attribute) =>
-      omit(attribute, ['aggregationMethods', 'grain', 'downSampleMethods'])
-    );
-  }
-  if (card.content?.series) {
-    series = card.content.series.map((attribute) =>
```

Comment:
There seems to be a lot of deletions on this page. Is this on purpose.

Reply:
Yes, these methods are been moved to CardEditor.jsx for bug fix